microsoft-edge/webview2/concepts/developer-guide.md

```diff
@@ -0,0 +1,80 @@
+---
+description: Learn about best practices to use when developing your WebView2 application.
+title: WebView2 development best practices  
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 03/04/2021
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+keywords: WebView2, webview2, WebView, webview, edge, best practices  
+---
+# WebView2 development best practices  
+
+Every development team follows different practices when building their application. When you build WebView2 applications, there are best practices we recommend you follow. This article outlines those recommendations and best practices for you when building production based WebView2 applications.
+
+
+## Use Evergreen WebView2 Runtime  
+
+We generally recommend using the Evergreen WebView2 Runtime instead of the Fixed Version Runtime.  The Evergreen WebView2 Runtime updates automatically and includes the latest features and security patches available to your WebView2 application. The Evergreen WebView2 Runtime also requires less storage space on the disk.
+
+Ensure the Evergreen WebView2 Runtime is installed before using your WebView2 application.  For more information, navigate to [Deploying the Evergreen WebView2 Runtime][Webview2ConceptsDistributionDeployingEvergreenWebview2Runtime].  
+
+## Run regular compatibility tests
+
+When using the Evergreen WebView2 Runtime, ensure you run regular compatibility tests. Because the runtime is automatically updated, you'll want to ensure that your WebView2 application performs as expected.  For more information, navigate to [Stay compatible in Evergreen mode][Webview2ConceptsDistributionStayCompatibleEvergreenMode].  
+
+## Ensure APIs are supported by the installed WebView2 Runtime
```

Comment:
+ @pagoe-msft here to take a look since he wrote the relevant doc for this in https://review.docs.microsoft.com/en-us/microsoft-edge/webview2/concepts/versioning?branch=user%2Fv-jomar7%2Fdocs%2FwebviewEnhancePractices20210302.